utils/getTrustedCertificates.ts

```diff
@@ -16,16 +16,27 @@ let _systemCertificates: (string | Buffer)[] | undefined;
 
 export async function getTrustedCertificates(): Promise<(string | Buffer)[]> {
     // tslint:disable-next-line:no-function-expression
```

Comment:
Would it be accurate to say that you expect most Windows/Mac users to use `docker.useCertificateStore` and only Linux users to use `docker.certificatePaths`? Just trying to understand the use-case for each and how common each case is